tools/setup-buildtools.sh

```diff
@@ -22,30 +25,12 @@ yum -y install devtoolset-7-valgrind
 
 yum-config-manager --enable rhel-server-rhscl-7-rpms
 
-if [ `gcc --version | grep 7` == "" ]; then
```

Comment:
Check for 7 was bad, and cmake installation got moved out into its own separate much smarter script right now.